xyz

```diff
@@ -119,8 +119,14 @@ next_version=$("$dir/node_modules/.bin/semver" -i "$increment" "$version") ||
 message="${message_template//X.Y.Z/$next_version}"
 tag="${tag_template//X.Y.Z/$next_version}"
 
-bold=$(tput bold)
-reset=$(tput sgr0)
+if [[ -t 1 ]] && [[ -n "$TERM" ]] && which tput &>/dev/null; then
```

Comment:
Had to `man type` :smile: 

That looks fine to me, I'll update the PR this evening